docs/core/logger.md

```diff
@@ -341,6 +341,18 @@ You can easily change the date format using one of the following parameters:
     --8<-- "examples/logger/src/date_formatting_output.json"
     ```
 
+### Environment variables
```

Comment:
In logger.md we already have a section that explains some ENV variables (https://docs.powertools.aws.dev/lambda/python/2.22.0/core/logger/#getting-started). How about adding a disclaimer like: "check for additional environment variables here" and adding a link to this new section?

Reply:
Yep, that makes sense. Will add it 👍
I've also seen that Metrics also mentions a few environment variables for the quick start, though it is only 1 environment variable so there are no other variables in the corresponding section.